obstacle_stop_planner_refine/include/obstacle_stop_planner/obstacle_point_cloud.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OBSTACLE_STOP_PLANNER__OBSTACLE_POINT_CLOUD_HPP_
+#define OBSTACLE_STOP_PLANNER__OBSTACLE_POINT_CLOUD_HPP_
+
+#include <memory>
+
+#include <rclcpp/logger.hpp>
+#include <sensor_msgs/msg/point_cloud2.hpp>
+#include "pcl/point_types.h"
+#include "pcl_conversions/pcl_conversions.h"
+#include "pcl/point_cloud.h"
+#include "pcl/common/transforms.h"
+#include "pcl/filters/voxel_grid.h"
+#include "tf2_ros/transform_listener.h"
+#include "autoware_planning_msgs/msg/trajectory.hpp"
+#include "obstacle_stop_planner/vehicle.hpp"
+
+namespace motion_planning {
+
+class ObstaclePointCloud
+{
+public:
+  explicit ObstaclePointCloud(rclcpp::Logger logger);
+
+  void SetPointCloud(const sensor_msgs::msg::PointCloud2::ConstSharedPtr msg);
+  void SetSearchRadius(const double value);
+  void SetVehicleInfo(const VehicleInfo vehicle_info);
```

Comment:
- Use camelCase (or snake_case is sometimes acceptable) for functions, PascalCase is only for classes.
- Use const reference for non-primitive types (in this case, VehicleInfo).
- In general, setters shouldn't be used. I'd like to consider removing and replacing them with something else.

Reply:
Thanks. Changed at ba16524

Reply:
`const VehicleInfo vehicle_info` should be `const VehicleInfo & vehicle_info`

Reply:
Already changed in other commits...? I'm not sure. Keeping this thread unresolved just in case.

Reply:
Yes, I've removed the setter function, so the latest code is fine.